lib/run_loop/app.rb

```diff
@@ -92,9 +116,62 @@ def sha1
 
     private
 
+    # @!visibility private
     def plist_buddy
       @plist_buddy ||= RunLoop::PlistBuddy.new
     end
 
+    # @!visibility private
+    # An otool factory.
+    def otool(file)
+      RunLoop::Otool.new(file)
+    end
+
+    # @!visibility private
+    # A strings factory
+    def strings(file)
+      RunLoop::Strings.new(file)
+    end
+
+    # @!visibility private
+    def skip_executable_check?(file)
+      image?(file) ||
+        plist?(file) ||
+        lproj_asset?(file) ||
+        code_signing_asset?(file)
+    end
```

Comment:
Could use a second pair of eyes on this.  Anything else I should skip?

Ack!  CoreData bundles!!!